src/hosts/BitbucketAdapter.js

```diff
@@ -88,14 +103,25 @@ const BitbucketAdapter = (config: Config) => {
         } merged  by Landkid after a successful build rebased on Master`,
         merge_strategy: 'merge_commit'
       };
-      Logger.info({ pullRequestId, endpoint }, 'Merging pull request');
```

Comment:
Not sure why prettier has decided to be stricter on these? We've always had prettier running...